lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -599,21 +635,50 @@ def get_stats(self, cookie: int = 0, cookie_mask: int = 0):
             self.logger.error("Could not obtain rule records due to either InvalidDatapath, OFError or UnexpectedMultiReply")
             return RuleRecordTable()
 
-def _generate_rule_match(imsi, ip_addr, rule_num, version, direction):
+    def _prepare_session_config_report(self, stats_msgs):
+        session_config_dict={}
+
+        for flow_stats in stats_msgs:
+            for stat in flow_stats:
+                if stat.table_id != self.tbl_num:
+                    return
+
+                local_f_teid_ng = _get_ng_local_f_id(stat)
+                if not local_f_teid_ng or local_f_teid_ng == REG_ZERO_VAL:
+                    continue
+                #Already present
+                if local_f_teid_ng in session_config_dict:
```

Comment:
How stats for multiple rules per local tied handled ?

Reply:
Yeah it looks like we would only store the first seen, which should be fine as we're only looking for session_version thats shared across flows @prabinakpattnaik ? 

Do we need some error handling where for some reason a few rules are on the old session_version?

Reply:
Changes done as requested.

Reply:
lgtm can you maybe just log an error here 
```
if local_f_teid_ng in session_config_dict:
   if local_f_teid_ng != session_config_dict[local_f_teid_ng].local_f_teid
        self.logger.error("Concerning... investigate.... smth like this")
      
   continue